### DISCUSSION TITLE
Select related fields in views like calendar

### DISCUSSION THREAD

**Discussion by lucameusburger** [directus/directus#13153] [Feature Requests]
A option to select related field in the calendar view would be really helpful.

Real example:
I use directus in event management and the data is structured like (really simplified):

event(name, time_start, time_end, day)
day(events, date_start, date_end)

So it's a n:1 relation. Now i want to use a calendar view for the events. But the event itself has no date, only the day has but there is no option to select related fields in the calendar view. In the list/table view there is.

<img width="611" alt="image" src="https://user-images.githubusercontent.com/25350597/167125411-3f7862a4-0ef9-4ff9-a1e7-eeb704115400.png">
Here you can see how the Start field is from the days table.

Keep it up directus community! Love it! ❤️

**Comment by benhaynes** [comment]
I agree — and can't think of any reason that relational fields shouldn't be available across all layout template config. 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
